ruby/ext/google/protobuf_c/map.c

```diff
@@ -388,6 +388,8 @@ VALUE Map_index(VALUE _self, VALUE key) {
 VALUE Map_index_set(VALUE _self, VALUE key, VALUE value) {
   Map* self = ruby_to_Map(_self);
```

Comment:
Should the frozen check happen before the conversion to Map?

Reply:
I doubt it really makes a difference, but I'll move them up just in case.